examples/aio-sx-hw-settle.yaml

```diff
@@ -96,7 +96,7 @@ spec:
   administrativeState: unlocked
   bootDevice: /dev/disk/by-path/pci-0000:00:0d.0-ata-1.0
   console: tty0
-  hwSettle: 0
+  hwSettle: "0"
```

Comment:
Would prefer to use a non-zero value in the example

Reply:
As discussed, default value is "0"